infrastructure/code/code.go

```diff
@@ -418,3 +507,15 @@ func (sc *Scanner) trackResult(success bool, scanMetrics *ScanMetrics) {
 		},
 	)
 }
+
+func (sc *Scanner) useIgnoresFlow() bool {
+	response, err := sc.SnykApiClient.FeatureFlagSettings(snyk_api.FeatureFlagSnykCodeConsistentIgnores)
+	if err != nil {
+		log.Debug().Err(err).Msg("Failed to check if the ignores experience is enabled")
+		return false
+	}
+	if !response.Ok && response.UserMessage != nil {
+		log.Info().Msg(*response.UserMessage)
+	}
+	return response.Ok
+}
```

Comment:
Just for discussion: `FeatureFlagSettings` wasn't a great choice of name (my bad 😢)
I think the `Settings` part does not make sense. Better if `FeatureFlagStatus`?

Reply:
I'm not very good at naming stuff but I guess you're right, `FeatureFlagStatus` is slightly better! I've renamed it